src/SignalR/common/Shared/TaskCache.cs

```diff
@@ -3,11 +3,11 @@
 
 using System.Threading.Tasks;
 
-namespace Microsoft.AspNetCore.SignalR.Internal
+namespace Microsoft.AspNetCore.Internal
```

Comment:
Nit: Why keep this under `/src/SignalR` but remove `SignalR` from the namespace? Is it used by non-SignalR projects?

Reply:
I get it now... It's used by Microsoft.AspNetCore.Http.Connections.